editions/tw5.com/tiddlers/community/Improving TiddlyWiki Documentation.tid

```diff
@@ -1,12 +1,21 @@
 created: 20140820151051019
-modified: 20150630205758258
+modified: 20150927205758258
 tags: Community
 title: Improving TiddlyWiki Documentation
 type: text/vnd.tiddlywiki
 
 Anyone can submit improvements to the TiddlyWiki documentation that appears on http://tiddlywiki.com.
+If you want to help, please start by reading the [[Documentation Style Guide]]. 
+
+Changes to the documentation can be submitted in two way: 
+
+* The Github method should be prefered if you are familiar with Github (or are happy to give it a try): see the detailed instructions and video tutorials below.
+* Otherwise, you can simply send the proposed content to the [[tiddlywikidocs discussion group|https://groups.google.com/forum/#!forum/tiddlywikidocs]], mentioning that you need someone to create the pull request for you. 
```

Comment:
I think it makes sense to use the documentation group for that and not only use it for doc-philosophy ...btw. there's a little grammatical glitch: "two **ways**"